gno.land/pkg/sdk/vm/msgs.go

```diff
@@ -86,6 +87,7 @@ func (msg MsgAddPackage) GetReceived() std.Coins {
 type MsgCall struct {
 	Caller  crypto.Address `json:"caller" yaml:"caller"`
 	Send    std.Coins      `json:"send" yaml:"send"`
+	Deposit std.Coins      `json:"deposit" yaml:"deposit"`
```

Comment:
Suggest "MaxDeposit" so it's clear it's not just sending coins, it's doing something and you get something back.
Also json omitempty so that users don't have to worry about it usually.